src/shared/src/components/Select.tsx

```diff
@@ -0,0 +1,71 @@
+import { makeStyles } from '@material-ui/core/styles';
+import cn from 'classnames';
+import * as React from 'react';
+import { AltinnAppTheme } from '../theme';
+import '../../../altinn-app-frontend/src/styles/shared.css';
+
+const useStyles = makeStyles({
+  select: {
+    fontSize: '1.6rem',
+    '&:focus': {
+      outline: `2px solid ${AltinnAppTheme.altinnPalette.primary.blueDark}`,
+    },
+  },
+});
+
+const optionStyle = {
+  display: 'none',
+};
+interface ISelectProps {
+  id: string;
+  onChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  onBlur?: (event: React.FocusEvent<HTMLSelectElement>) => void;
+  options: { value: string; label: string }[];
+  value: string;
+  disabled?: boolean;
+  error?: boolean;
+  fullWidth?: boolean;
+  className?: string;
+}
+
+const Select = ({
+  id,
+  onChange,
+  onBlur,
+  options,
+  value,
+  disabled = false,
+  error = false,
+  fullWidth = false,
+  className,
+}: ISelectProps) => {
+  const classes = useStyles();
+  return (
+      <select
+        id={id}
+        value={value}
+        disabled={disabled}
+        className={cn(
+          classes.select,
+          'custom-select a-custom-select',
+          {
+            'validation-error': error,
+            'disabled !important': disabled,
+          },
+          className,
+        )}
+        style={{ width: fullWidth ? '100%' : 'unset' }}
+        onChange={onChange}
+        onBlur={onBlur}
+      >
+        <option style={optionStyle} />
+        {options.map((option, index) => (
+          <option key={index} value={option.value}>
```

Comment:
Using index as key isn't recommended.
You could use the `option.value` or `option.label` instead.
It would make sense as a select with two equal values would be quite confusing.
WDYT?
If you keep the firs option, on line 61, you could give it a key as well. Maybe something describing intent?

Reply:
jupp, option.value is better